Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -15,26 +15,12 @@
     <% if root_url != "#{HOMEBREW_BOTTLE_DEFAULT_DOMAIN}/bottles" %>
     root_url "<%= root_url %>"
     <% end %>
-    <% if ![HOMEBREW_DEFAULT_PREFIX,
-            HOMEBREW_MACOS_ARM_DEFAULT_PREFIX,
-            HOMEBREW_LINUX_DEFAULT_PREFIX].include?(prefix) %>
-    prefix "<%= prefix %>"
```

Comment:
Are we dropping `prefix` method in bottle blocks?

Reply:
I think `prefix` is gone in favor of `cellar`.

Reply:
Woups I removed the prefix by mistake. The goal this PR is not to touch the prefix part, that can be done in a separate PR. I'll add it back.